tasks/release.py

```diff
@@ -906,3 +906,39 @@ def chase_release_managers(_, version):
     client = WebClient(os.environ["SLACK_API_TOKEN"])
     for channel in channels:
         client.chat_postMessage(channel=channel, text=message)
+
+
+@task
+def check_for_changes(ctx, release_branch, warning_mode=False):
+    """
+    Check if there was any modification on the release repositories since last release candidate.
+    """
+    next_version = next_rc_version(ctx, "7")
+    repo_data = generate_repo_data(warning_mode, next_version, release_branch)
+    changes = 'false'
+    for repo_name, repo in repo_data.items():
+        head_commit = get_last_commit(ctx, repo_name, repo['branch'])
+        last_tag_commit, last_tag_name = get_last_tag(ctx, repo_name, next_version.tag_pattern())
+        if last_tag_commit != "" and last_tag_commit != head_commit:
+            changes = 'true'
+            print(f"{repo_name} has new commits since {last_tag_name}", file=sys.stderr)
+            if warning_mode:
+                rm = release_manager(repo_name, repo['branch'])
```

Comment:
`release_manager` yields the emails but we use it as if it was return just an email. Is it expected?

Reply:
Nice catch